docs/core/tools/dotnet-new-details.md

```diff
@@ -0,0 +1,69 @@
+---
+title: dotnet new details
+description: The dotnet new details command displays template package metadata.
+ms.date: 06/07/2023
+---
+# dotnet new details
+
+**This article applies to:** ✔️ .NET 8 preview 6
+
+## Name
+
+`dotnet new details` - Displays template package metadata.
+
+## Synopsis
+
+```dotnetcli
+dotnet new details [<PACKAGE_NAME>] [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [--force] [-d|--diagnostics] [-h|--help]
+```
+
+## Description
+
+The `dotnet new details` command displays the metdata of the template package from the package name provided. For information on a specific version specify the version in with the `--version (check if this is correct)` option. By default, the command searches for the latest available version.
+If the package is installed locally or is found on the official NuGet website, it also displays the templates that the package contains, otherwise it will only display basic metadata.
+
+## Arguments
+
+- **`PACKAGE NAME`**
+
+ The package identifier to display the details for.
+
+## Options
+
+- **`--add-source|--nuget-source <SOURCE>`**
+  
+  By default, `dotnet new details` uses the hierarchy of NuGet configuration files from the current directory to determine the NuGet source the package can be installed from. If `--nuget-source` is specified, the source will be added to the list of sources to be checked.  
+  To check the configured sources for the current directory use [`dotnet nuget list source`](dotnet-nuget-list-source.md). For more information, see [Common NuGet Configurations](/nuget/consume-packages/configuring-nuget-behavior)
+
+[!INCLUDE [interactive](../../../includes/cli-interactive-5-0.md)]
```

Comment:
No need to specify 5.0 for the option since the command itself is later than that.
```suggestion
[!INCLUDE [interactive](../../../includes/cli-interactive.md)]
```